pkg/annotations/constants.go

```diff
@@ -50,39 +50,40 @@ const (
+	SvcLBSuffixProxyProtocol                             = "aws-load-balancer-proxy-protocol"
+	SvcLBSuffixIPAddressType                             = "aws-load-balancer-ip-address-type"
+	SvcLBSuffixAccessLogEnabled                          = "aws-load-balancer-access-log-enabled"
+	SvcLBSuffixAccessLogS3BucketName                     = "aws-load-balancer-access-log-s3-bucket-name"
+	SvcLBSuffixAccessLogS3BucketPrefix                   = "aws-load-balancer-access-log-s3-bucket-prefix"
+	SvcLBSuffixCrossZoneLoadBalancingEnabled             = "aws-load-balancer-cross-zone-load-balancing-enabled"
+	SvcLBSuffixSSLCertificate                            = "aws-load-balancer-ssl-cert"
+	SvcLBSuffixSSLPorts                                  = "aws-load-balancer-ssl-ports"
+	SvcLBSuffixSSLNegotiationPolicy                      = "aws-load-balancer-ssl-negotiation-policy"
+	SvcLBSuffixBEProtocol                                = "aws-load-balancer-backend-protocol"
+	SvcLBSuffixAdditionalTags                            = "aws-load-balancer-additional-resource-tags"
+	SvcLBSuffixHCHealthyThreshold                        = "aws-load-balancer-healthcheck-healthy-threshold"
+	SvcLBSuffixHCUnhealthyThreshold                      = "aws-load-balancer-healthcheck-unhealthy-threshold"
+	SvcLBSuffixHCTimeout                                 = "aws-load-balancer-healthcheck-timeout"
+	SvcLBSuffixHCInterval                                = "aws-load-balancer-healthcheck-interval"
+	SvcLBSuffixHCProtocol                                = "aws-load-balancer-healthcheck-protocol"
+	SvcLBSuffixHCPort                                    = "aws-load-balancer-healthcheck-port"
+	SvcLBSuffixHCPath                                    = "aws-load-balancer-healthcheck-path"
+	SvcLBSuffixHCSuccessCodes                            = "aws-load-balancer-healthcheck-success-codes"
+	SvcLBSuffixTargetGroupAttributes                     = "aws-load-balancer-target-group-attributes"
+	SvcLBSuffixSubnets                                   = "aws-load-balancer-subnets"
+	SvcLBSuffixEIPAllocations                            = "aws-load-balancer-eip-allocations"
+	SvcLBSuffixPrivateIpv4Addresses                      = "aws-load-balancer-private-ipv4-addresses"
+	SvcLBSuffixIpv6Addresses                             = "aws-load-balancer-ipv6-addresses"
+	SvcLBSuffixALPNPolicy                                = "aws-load-balancer-alpn-policy"
+	SvcLBSuffixTargetNodeLabels                          = "aws-load-balancer-target-node-labels"
+	SvcLBSuffixLoadBalancerAttributes                    = "aws-load-balancer-attributes"
+	SvcLBSuffixLoadBalancerSecurityGroups                = "aws-load-balancer-security-groups"
+	SvcLBSuffixManageSGRules                             = "aws-load-balancer-manage-backend-security-group-rules"
+	SvcLBSuffixEnforceSGInboundRulesOnPrivateLinkTraffic = "aws-load-balancer-inbound-sg-rules-on-private-link-traffic"
```

Comment:
should this be something like
```
aws-load-balancer-enforce-sg-inbound-rules-on-pl-traffic
```
or 
```
aws-load-balancer-enforce-sg-inbound-rules-on-private-link-traffic
```

Reply:
Good point,  I like `aws-load-balancer-enforce-sg-inbound-rules-on-private-link-traffic` but annotation must be 63 characters or less, so we decided to use `aws-load-balancer-inbound-sg-rules-on-private-link-traffic`